easybuild/easyconfigs/b/beagle-lib/beagle-lib-3.1.2-GCC-10.3.0-CUDA-11.3.1.eb

```
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'beagle-lib'
+version = '3.1.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/beagle-dev/beagle-lib'
+description = """beagle-lib is a high-performance library that can perform the core calculations at the heart of most
+ Bayesian and Maximum Likelihood phylogenetics packages."""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/beagle-dev/beagle-lib/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['dd872b484a3a9f0bce369465e60ccf4e4c0cd7bd5ce41499415366019f236275']
+
+dependencies = [
+    ('CUDA', '11.3.1', '', True),
+    ('Java', '17', '', True),
```

Unless Java 17+ is required, it would be good to stick to the "default" Java version used in that toolchain generation
```suggestion
    ('Java', '11', '', True),
```